1_Mechanics/02_GMGuide.md

```diff
@@ -7,17 +7,19 @@
 
```

Comment:
I'm looking at commit 049f2e5. In the future, please separate hard wrap into separate commits that I can skip if I'm reviewing commit-by-commit

Reply:
Will do!